content/map/6/3/readme.md

```diff
@@ -68,7 +68,7 @@ es:
     reducirían notablemente el tiempo necesario para una intervención médica
     efectiva. La página web de “Enfermesoma” ofrece un explorador de relaciones
     interactivas entre enfermedades y enlaces a documentos y bases de datos en
-    línea en http://diseasome.eu.
+    línea en ![http://diseasome.eu](http://diseasome.eu).
```

Comment:
Pretty sure adding a ! makes it an image...

Reply:
@lecross

Reply:
That was the only trick I found for making a url into text. If there is another method, let me know. I'll check with Lisel about removing the link entirely.

Reply:
Use `<http://diseasome.eu>`